content/post/055-js-libp2p-0.26.md

```diff
@@ -0,0 +1,128 @@
+---
+date: 2019-08-07
+url: 2019-08-07-js-libp2p-0-26
+title: js-libp2p 0.26.0 released
+author: Jacob Heun
+---
+
+> Gossipsub support and Promisify API
+
+# 🔦 Highlights
+
+## 🗣 Gossipsub
+Thanks to the awesome work of the [ChainSafe](https://github.com/ChainSafe) team, [Gossipsub](https://github.com/ChainSafe/gossipsub-js) is here! If you are using Pubsub you can now switch to using Gossipsub instead of Floodsub. Have old peers you need to flood stuff to? Don't worry, Gossipsub will automatically fallback to Floodsub for peers that don't support it. See the API Changes section below for how to migrate your config over to using Gossipsub.
```

Comment:
What is the difference, why is gossipsub better?

Reply:
Added some brief text explaining the main difference and linked to the Spec that has a much more detailed overview.